src/exprs.c

```diff
@@ -954,10 +954,10 @@ Obj             EvalPermExpr (
 			    c, MAX_DEG_PERM4);
 
             /* if necessary resize the permutation                         */
-            if ( SIZE_OBJ(perm)/sizeof(UInt4) < c ) {
-                ResizeBag( perm, (c + 1023) / 1024 * 1024 * sizeof(UInt4) );
-                ptr4 = ADDR_PERM4( perm );
-                for ( k = m+1; k <= SIZE_OBJ(perm)/sizeof(UInt4); k++ ) {
+            if ( DEG_PERM4(perm) < c ) {
+	      ResizeBag( perm, SIZEBAG_PERM4((c + 1023) / 1024 * 1024) );
+	      ptr4 = ADDR_PERM4( perm );
+	      for ( k = m+1; k <= DEG_PERM4(perm); k++ ) {
```

Comment:
In the GitHub diff view, the indentation here seems to get worse with this PR...

Reply:
These are tabs, not spaces (did we somewhere say that we prefer spaces for formatting?), the same goes for line 2178 in `intrprtr.c`.